mk/fpgaaccel.mk

```diff
@@ -0,0 +1 @@
+CFLAGS += -DENABLE_FPGA_ACCEL
```

Comment:
You can set CFLAGS in Makefile. The reason why `mk/fpga-accel.mk` exists is to provide modularized build procedures.